_integration-schemas/shopify/customers.md

```diff
@@ -6,7 +6,7 @@ name: "customers"
 doc-link: "https://shopify.dev/docs/admin-api/rest/reference/customers"
 singer-schema: "https://github.com/singer-io/tap-shopify/blob/master/tap_shopify/schemas/customers.json"
 description: |
-  The `{{ table.name }}` table contains info about the shop's customers. This includes their contact details, order history, and email marketing preferences.
+  The `{{ table.name }}` table contains info about the shop's customers. This includes their contact details, order history, and email marketing preferences. Null values are now accepted for replication.
```

Comment:
I think we updated the changelog with this last month: https://github.com/singer-io/tap-shopify/pull/94/files

I don't think we need to explicitly mention this. Typically we'll allow `null` values for replication, with the exceptions potentially being Primary and Replication Keys.